### ISSUE TITLE
I want to know how many epoches do I have to run..

### ISSUE THREAD

**Issue by e05087** [YerevaNN/R-NET-in-Keras#4]
Hello, I'm ml student and I'm studying about R-net. While I doing that, I ran this code about 10 epoches, 
and I get only 10-13% acc. And loss doesn't go down anymore when it reaches 16.xx~. I think it is not right result that you said. I want to know where I miss..
Because of hardware limit, I ran with glove - 50 dim, and batch size - 30. And other things same.
Can you let me know how many epoches do I have to run until I get your result?

**Comment by mahnerak** [comment]
@e05087 May you try again with updated instructions?
Also, consider choosing Theano as backend, it may help you speed up training process.
We got EM=57.52% and F1=67.42% with model trained about 30 epochs.
